src/Integration.Vsix.UnitTests/SonarLintTagger/CancellableJobRunnerTests.cs

```diff
@@ -129,7 +114,15 @@ public void CancelAfterFirstOperation()
                 testLogger.WriteLine("[Test] Executing op1");
                 op1Executed = true;
 
-                testSubject.Cancel();
+                // in some cases, the op1 is executed before the CancellableJobRunner.Start returned the runner, so testSubject can be null
```

Comment:
Could put a ManualResetEvent here and release it after Start has returned (and before waiting for it). It took me a while to understand this change